src/System.Runtime.Environment/src/RuntimeInfo.Linux.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    public static class RuntimeInfo
```

Comment:
Could we use RuntimeEnvironment? but if you prefer, I am fine with RuntimeInformation. It just sounds like RuntimeInfo, but expanded to follow the naming conventions.